packages/common/src/store/playback-position/types.ts

```diff
@@ -10,5 +11,7 @@ export type PlaybackPositionInfo = {
 }
 
 export type PlaybackPositionState = {
-  trackPositions: { [Key in ID]?: PlaybackPositionInfo }
+  [Key in ID]?: {
+    trackPositions: { [Key in ID]?: PlaybackPositionInfo }
+  }
```

Comment:
Don't need to fix here, but wondering if we should start naming these better to make the types more readable:

```
export type PlaybackPositionState = {
  [UserIdKey in ID]?: {
    trackPositions: { [TrackIdKey in ID]?: PlaybackPositionInfo }
  }
```